### PR TITLE
Added note on excluding pods from pod preset per trello

### PR DESCRIPTION
**Pull request by mburke5678**
https://trello.com/c/BikBRkoc/502-document-catalog-ability-to-restrict-the-set-of-pods-a-podpreset-can-target

### CODE REVIEW

**Comment by mburke5678** [comment]
Added statement on using flag to exclude pods from pod preset.
Added pod preset information to Arch -> Core Concepts -> Pods and Services
Added pod preset information to  Admin Guide -> Managing Pods

**Comment by mburke5678** [comment]
@jpeeler PTAL

**Review by jpeeler — APPROVED** [review]
Generally I think I prefer the wording of "pod spec not being altered by any podpreset mutations" over "injected". But I don't have strong feelings on it.

**Comment by wmengRH** [comment]
LGTM. Thanks.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: architecture/core_concepts/pods_and_services.adoc [hunk 1]
```diff
@@ -161,6 +161,49 @@ link:https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod docu
...
+
+Using pod preset objects you can inject:
+
+* xref:../../dev_guide/secrets.adoc#dev-guide-secrets[secret objects]
+* xref:../../dev_guide/configmaps.adoc#consuming-configmap-in-pods[`ConfigMap` objects]
+* xref:../../dev_guide/volumes.adoc#dev-guide-volumes[storage volumes]
+* container volume mounts
+* environment variables
+
+Developers only need make sure the pod labels match the label selector on the PodPreset in order to add all that information to the pod. The xref:../../architecture/core_concepts/pods_and_services.adoc#labels[label] on a pod associates the pod with one or more pod preset objects that have a matching xref:../../architecture/core_concepts/pods_and_services.adoc#services[label selectors].
```
Comment (ahardin-rh) [inline architecture/core_concepts/pods_and_services.adoc]: s/Developers only need make sure the/Developers need to ensure the

File: architecture/core_concepts/pods_and_services.adoc [hunk 2]
```diff
@@ -161,6 +161,49 @@ link:https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod docu
...
+
+Using pod presets, a developer can provision pods without needing to know the details about the services the pod will consume. An administrator can keep configuration items of a service invisible from a developer without preventing the developer from deploying pods.
```
Comment (ahardin-rh) [inline architecture/core_concepts/pods_and_services.adoc]: Is there extra spacing before "An administrator"?

Comment (mburke5678) [reply architecture/core_concepts/pods_and_services.adoc]: No.

**Pull request merged**